.gitignore

```diff
@@ -45,7 +45,7 @@ dev/create-release/*final
 dev/create-release/*txt
 dev/pr-deps/
 dist/
-docs/_site
+**/docs/_site/
```

Comment:
Without this change, git was not ignoring `python/docs/_site`.

Reply:
That's fine, or just explicitly specify python/docs/_site (is there another?)

Reply:
I'm not aware of any others, so I can update it to just mention Python specifically on a separate line.